conversation_test.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"accounts-service/auth"
+	"accounts-service/models/memory"
+	accountsv1 "accounts-service/protorepo/noted/accounts/v1"
+	"context"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/suite"
+)
+
+type ConversationsAPISuite struct {
+	suite.Suite
+	// account *accountsAPI
+	auth auth.TestService
+	srv  *conversationsAPI
+}
+
+func TestConversationsService(t *testing.T) {
+	suite.Run(t, &ConversationsAPISuite{})
+}
+
+func (s *ConversationsAPISuite) SetupSuite() {
+	logger := newLoggerOrFail(s.T())
+	db := newDatabaseOrFail(s.T(), logger)
```

Comment:
Can you please rename this to `newMemoryTestDatabaseOrFail`? So it's less ambiguous.